tests/dynamic_friction.prm

```diff
@@ -47,20 +35,26 @@ subsection Mesh deformation
 end
 
 subsection Material model
-
-  set Model name = dynamic friction
-  subsection Dynamic Friction
+  set Model name = visco plastic
+  subsection Visco Plastic
     set Densities         = 2800
     set Viscosity averaging scheme = harmonic
-    subsection Viscosities
-       set Minimum viscosity = 1e19
-       set Maximum viscosity = 1e25
-       set Reference strain rate = 1e-15
-       set Background Viscosities = 1.e25, 1.e20
-       set Coefficients of static friction = 0.5547 , 0.5
-       set Coefficients of dynamic friction = 0.6 , 0.5
-       set Cohesions = 20e6
-    end
+    set Minimum viscosity = 1e19
+    set Maximum viscosity = 1e25
+    set Reference strain rate = 1e-15
+
+    # Dislocation creep parameters for
+    set Prefactors for dislocation creep          =      1e-20, 1e-25
+    set Stress exponents for dislocation creep    =      1.0
+    set Activation energies for dislocation creep =      0.0
+    set Activation volumes for dislocation creep  =      0.0
+
+  # Plasticity parameters
+  set Friction dependence mechanism       = dynamic friction
+  set Dynamic characteristic strain rate  = 1e-15
```

Comment:
What about the smoothing exponent? Is it by default not applied?

Reply:
smoothing exponent is 1 by default, so it is not applied